releasenotes/notes/branding-de80e7d9c1392e69.yaml

```diff
@@ -0,0 +1,9 @@
+---
+fixes:
+  - This release fixes a race condition where the
+    branding could be overriden depending on the playbook
+    execution order. Deployers have to make sure the
```

Comment:
Based on the conversation in #1088 this doesn't even sound like the right set of instructions.

Also, shouldn't this be described in a known-issue section? I feel like this is the description of a fix for a known bug that is not fixed.